src/views/Login/Ticket.js

```diff
@@ -135,6 +137,8 @@ export default function Ticket({ className, goHome }) {
       point: impliedPoint || '',
       usePasshrase: false,
       useShards: false,
+      showTicket: true,
```

Comment:
wait, I missed this, but shouldn't this be default `false`, as per common password field convention?

Reply:
the website mentions that:

> When offered the choice of masked or unmasked passwords within the interface, the concept was identified as a feature not an error. Participants appreciated the usability benefits of clear text passwords and the security benefits of masked passwords. When participants used this option their trustingness of the website was unaffected. (Note the password was unmasked by default)

Reply:
Oh I hadn't remembered that note at the end. Very interesting. This is probably fine then.